cypress/support/fragments/finance/funds/funds.js

```diff
@@ -115,10 +117,25 @@ export default {
     cy.xpath(viewTransactionsLinkXpath).click();
   },
 
-  checkTransaction: (value, fundCode) => {
-    // TODO: refactor using interactors (Mutli column list)
-    cy.expect(Pane({ id: transactionResultPaneId }).find(HTML(including('$' + value.toFixed(2)))).exists());
-    cy.expect(Section({ id: transactionResultPaneId }).find(HTML(including(fundCode))).exists());
+  checkTransaction: (rowNumber, transaction) => {
+    cy.expect(Pane({ id: transactionResultPaneId })
+      .find(MultiColumnListRow({ index: rowNumber }))
+      .find(MultiColumnListCell({ content: transaction.from })).exists());
+    cy.expect(Pane({ id: transactionResultPaneId })
+      .find(MultiColumnListRow({ index: rowNumber }))
+      .find(MultiColumnListCell({ content: transaction.to })).exists());
+    cy.expect(Pane({ id: transactionResultPaneId })
+      .find(MultiColumnListRow({ index: rowNumber }))
+      .find(MultiColumnListCell({ content: transaction.source })).exists());
+    cy.expect(Pane({ id: transactionResultPaneId })
+      .find(MultiColumnListRow({ index: rowNumber }))
+      .find(MultiColumnListCell({ content: transaction.type })).exists());
+    cy.expect(Pane({ id: transactionResultPaneId })
+      .find(MultiColumnListRow({ index: rowNumber }))
+      .find(MultiColumnListCell({ content: transaction.amount })).exists());
+    cy.expect(Pane({ id: transactionResultPaneId })
+      .find(MultiColumnListRow({ index: rowNumber }))
+      .find(MultiColumnListCell({ content: transaction.tags })).exists());
```

Comment:
May be better use for in? May be not, just suggestion :)

```
const paneRowNumber = Pane({ id: transactionResultPaneId }).find(MultiColumnListRow({ index: rowNumber }));

for (let prop in transaction) {
  cy.expect(paneRowNumber.find(MultiColumnListCell({ content: prop })).exists());
}
```

Reply:
Agree, will update